src/Misc.ino

```diff
@@ -2551,6 +2551,10 @@ boolean ruleMatch(String& event, String& rule)
   String tmpEvent = event;
   String tmpRule = rule;
 
+  //Ignore escape char
+  tmpRule.replace("[","");
+  tmpRule.replace("]","");
+
```

Comment:
When replacing a single character in a String, please use `'['` instead of `"["`
It is the difference between a char and a String. And String without the `F( )` macro are stored in RAM, not in flash.
I removed a lot of them last few days to save RAM.
Note that you can only replace a char with a char and the same for Strings.  (not entirely sure about acceptance of `''` in the replace function)